packager/react-packager/src/DependencyResolver/DependencyGraph/ResolutionRequest.js

```diff
@@ -226,7 +226,7 @@ class ResolutionRequest {
         realModuleName => {
           const searchQueue = [];
           for (let currDir = path.dirname(fromModule.path);
-               currDir !== '/';
+               currDir !== path.parse(fromModule.path).root;
```

Comment:
The result of `path.parse` is constant and this is being evaluated on each iteration so it makes sense to hoist this.